package.json

```diff
@@ -7,6 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "gulp"
   },
+  "engines": {
+    "node": "12"
```

Comment:
Do we know what node version cdn.threesixtygiving.org runs? @drkane ?

Reply:
Not sure - it's whatever the default for netlify is

Reply:
Looks like it's building fine on netlify with this package.json